WordPress/Classes/ViewRelated/NUX/SigninWPComSyncHandler.swift

```diff
@@ -0,0 +1,88 @@
+import UIKit
+import WordPressComAnalytics
+
+/// A protocol and extension encapsulating syncing behavior common to WPCom 
+/// signin controllers.  Responsible for syncing account information and blog 
+/// details of the user.
+///
+protocol SigninWPComSyncHandler: class
+{
+    func configureLoading(loading: Bool)
+    func configureStatusMessage(message: String)
```

Comment:
I'd prefer to keep with the "setupMethodName" and "configureMethodName" pattern used within the signin vcs. Rather than making them look more like setters how about we make them look less like setters.